examples/sumologic.yaml

```diff
@@ -20,6 +20,13 @@ extensions:
   file_storage:
     directory: /var/lib/otelcol-sumo/file_storage
 
+  ## Configuration for OpAMP Extension
+  ## The OpAMP extension remotely manages collector configuration.
+  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/extension/opampextension
+  opamp:
+    remote_configuration_directory: /etc/otelcol-sumo/conf.d
```

Comment:
Currently using the existing `conf.d` directory as the remote configuration directory. The OpAMP agent will overwrite its contents with received remote configuration. Alternatively, we could create a `remote.d` or `opamp.d` directory for this purpose. WDYT?